README.md

```diff
@@ -21,6 +21,19 @@ If you want to install [`clickhouse-cityhash`](https://pypi.org/project/clickhou
 > pip install asynch[compression]
 ```
 
+## Release v0.3.0 announcement
+
+The version 0.2.5 should have been named v0.3.0 due to compatibility-breaking changes.
+Since the release v0.2.5 is already yanked (of sorts), a smooth upgrade seems inexpedient.
+
+Before upgrade to the v0.3.0, please pay attention to the incompatible changes:
+
+1. the `connect` function from the `async/connection.py` module is not asynchronous context manager.
```

Comment:
is `now` asynchronous, I guess

Reply:
Thank you, fixed, please check and resolve this conversation.